ajax/libs/Leaflet.EasyButton/package.json

```diff
@@ -0,0 +1,30 @@
+{
+  "name": "leaflet-easybutton",
+  "description": "easily add control buttons to your leaflet maps with icon support",
+  "main": "easy-button.js",
```

Comment:
Please remove `main`, `style`, `scripts`, `bugs` and `dependencies` since we don't need them. Thank you.